forge-gui/res/cardsfolder/upcoming/mocking_doppelganger.txt

```diff
@@ -0,0 +1,9 @@
+Name:Mocking Doppelganger
+ManaCost:3 U
+Types:Creature Shapeshifter
+PT:0/0
+K:ETBReplacement:Copy:DBCopy:Optional
+K:Flash
+SVar:DBCopy:DB$ Clone | Choices$ Creature.OppCtrl | AddStaticAbilities$ FamilyTease | SpellDescription$ You may have CARDNAME enter the battlefield as a copy of a creature an opponent controls, except it has "Other creatures with the same name as this creature are goaded."
+SVar:FamilyTease:Mode$ Continuous | Affected$ Creature.sameName+Other | Goad$ True | Description$ You may have CARDNAME enter the battlefield as a copy of a creature an opponent controls, except it has "Other creatures with the same name as this creature are goaded."
```

Comment:
Change to `Description$ Other creatures with the same name as this creature are goaded.`